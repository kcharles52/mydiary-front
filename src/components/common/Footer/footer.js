import React from "react";
import { CardFooter } from "reactstrap";

export const Footer = () => (
  <CardFooter className="footer" color="secondary">
    <p> Designed by Charles, Copyright Â© 2018</p>
  </CardFooter>
);
export default Footer;