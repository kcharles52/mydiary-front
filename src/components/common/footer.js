import React from "react";
import { CardFooter } from "reactstrap";

const Footer = () => (
  <CardFooter className="footer" color="secondary">
    <p> Designed by Charles, Copyright Â© 2018</p>
  </CardFooter>
);
export default Footer;