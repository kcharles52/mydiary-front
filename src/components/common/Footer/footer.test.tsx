// react libraries
import * as React from "react";

// third party libraries
import { shallow } from "enzyme";

//components
import { Footer } from "./footer";

describe("Render footer", () => {
  let wrapper, shallowWrapper;
  beforeEach(() => {
    shallowWrapper = shallow(<Footer />);
  });
  afterEach(() => {
    shallowWrapper = null;
  });

  it("should render the footer correctly", () => {
    expect(shallowWrapper).toMatchSnapshot();
  });
});
