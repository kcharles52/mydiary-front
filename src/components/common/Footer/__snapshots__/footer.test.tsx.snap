// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render footer should render the footer correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <p>
         Designed by Charles, Copyright © 2018
      </p>,
      "className": "footer",
      "color": "secondary",
      "tag": "div",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Designed by Charles, Copyright © 2018",
      },
      "ref": null,
      "rendered": " Designed by Charles, Copyright © 2018",
      "type": "p",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <p>
           Designed by Charles, Copyright © 2018
        </p>,
        "className": "footer",
        "color": "secondary",
        "tag": "div",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Designed by Charles, Copyright © 2018",
        },
        "ref": null,
        "rendered": " Designed by Charles, Copyright © 2018",
        "type": "p",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
